Admin.js

```diff
@@ -34,78 +35,99 @@ var Admin = {
         Admin.setup_sticky_elements(subject);
         Admin.setup_readmore_elements(subject);
 
-//        Admin.setup_list_modal(subject);
+        //        Admin.setup_list_modal(subject);
+
     },
-    setup_list_modal: function(modal) {
-        Admin.log('[core|setup_list_modal] configure modal on', modal);
-        // this will force relation modal to open list of entity in a wider modal
-        // to improve readability
-        jQuery('div.modal-dialog', modal).css({
-            width:  '90%', //choose your width
-            height: '85%',
-            padding: 0
+    "setup_list_modal" (modal) {
```

Comment:
- eslint: Expected object keys to be in ascending order. 'setup_list_modal' should be before 'shared_setup'.